app/src/main/java/io/github/droidkaigi/confsched2018/presentation/sessions/SessionsFragment.kt

```diff
@@ -147,6 +161,20 @@ class SessionsFragment : Fragment(), Injectable, Findable, OnReselectedListener
         )
     }
 
+    override fun onPause() {
```

Comment:
Probably currently this method not called when changing to other bottom navigation.
Probably you can use `setUserVisibleHint()`

Reply:
Uhmm...
It seems to me that it is called in the timing.
I try the `setUserVisibleHint()` too.

Reply:
Sorry if it work well, You don't have to change it 🙇

Reply:
Thank you for checking! 🙇